### PR TITLE
[IE TESTS] Move ExecGraphInfoTests 

### PR DESCRIPTION
**Pull request by antonzaycev96**

### CODE REVIEW

File: inference-engine/tests/functional/plugin/gpu/shared_tests_instances/behavior/exec_graph_info.cpp [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+using namespace LayerTestsDefinitions;
+
+namespace {
+    const std::vector<InferenceEngine::Precision> netPrecisions = {
+            InferenceEngine::Precision::FP32,
+            InferenceEngine::Precision::FP16
+    };
+
+    const std::vector<std::map<std::string, std::string>> configs = {
+            {},
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/gpu/shared_tests_instances/behavior/exec_graph_info.cpp:18]: What about multi?

File: inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp [hunk 2]
```diff
@@ -17,5 +17,8 @@ std::vector<std::string> disabledTestPatterns() {
             ".*ActivationLayerTest\\.CompareWithRefs/Log.*netPRC=FP32.*",
             ".*ActivationLayerTest\\.CompareWithRefs/Sigmoid.*netPRC=FP32.*",
             ".*ActivationLayerTest\\.CompareWithRefs/Relu.*netPRC=FP32.*",
+
+            // TODO: currently this tests are not applicable to myriadPlugin
+            ".*Behavior.*ExecGraphTests.*"
```
Comment (iefode) [inline inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp:22]: Is it disabled in old infra?

File: inference-engine/tests/functional/plugin/gpu/shared_tests_instances/behavior/exec_graph_info.cpp (hunk 1 above)
Comment (antonzaycev96) [reply inference-engine/tests/functional/plugin/gpu/shared_tests_instances/behavior/exec_graph_info.cpp:18]: missing in the old infra.

File: inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp (hunk 2 above)
Comment (antonzaycev96) [reply inference-engine/tests/functional/plugin/myriad/shared_tests_instances/skip_tests_config.cpp:22]: Yes

**Comment by azhogov** [comment]
ci/jenkin passed with internal check

**Pull request merged**